Fixed an issue where format sql was messing up operator. #5083

pravesh-sharma opened the pull request:
- Also fixes https://github.com/pgadmin-org/pgadmin4/issues/6785, https://github.com/pgadmin-org/pgadmin4/issues/6990, https://github.com/pgadmin-org/pgadmin4/issues/7115.
- Using new library sql-formattor for SQL formatting.
- Added new options data type case, expression width, function case, lines between queries, logical operator new line and new line before semicolon for SQL formatting.
- Removed old options comma-first notations, re-indent aligned, re-indent, strip comments and wrap after N characters as not available in the new library.
- Capitalize casing has been removed as it was not supported by the new library using preserve casing instead.

adityatoshniwal reviewed (changes requested):
1. The backend API is not cleaned up.
2. The un-used lib is not removed from requirements.txt.
3. Commit message has no details on what new features are added and what removed in detail.
4. The commit message doesn't mention all other formatting related issues which will be fixed with this commit.
5. The commit message doesn't mention which lib is removed and which is added.

pravesh-sharma commented:
> 1. The backend API is not cleaned up.
> 2. The un-used lib is not removed from requirements.txt.
> 3. Commit message has no details on what new features are added and what removed in detail.
> 4. The commit message doesn't mention all other formatting related issues which will be fixed with this commit.
> 5. The commit message doesn't mention which lib is removed and which is added.
> 
> This is a very lazy PR.

I have removed the backend API. Also, I updated the commit. I did not mention the other issues in the commit as they were already linked in the original issue I have fixed. However, I have mentioned them now.
